src/main/java/nextstep/courses/domain/Image.java

```diff
@@ -0,0 +1,37 @@
+package nextstep.courses.domain;
+
+public class Image {
+    private String fileName;
+    private String contentType;
+    private long sizeInBytes;
+    private int width;
+    private int height;
+
+    public Image(String fileName, String contentType, long sizeInBytes, int width, int height) {
+        this.fileName = fileName;
+        this.contentType = contentType;
+        this.sizeInBytes = sizeInBytes;
+        this.width = width;
+        this.height = height;
+        validate();
```

Comment:
```suggestion
        validate();
        this.fileName = fileName;
        this.contentType = contentType;
        this.sizeInBytes = sizeInBytes;
        this.width = width;
        this.height = height;
```
결과적으로는 상관없지만 어차피 검증을 해야한다면 먼저 검증하는것이 맞지 않을까요?